kernel/src/acpi/tables.rs

```diff
@@ -182,8 +183,8 @@ impl ACPITable {
     ///
     /// This method returns the 4-character signature of the ACPI table, such as "APIC."
     #[expect(dead_code)]
-    fn signature(&self) -> FixedString<4> {
-        FixedString::from(self.header.sig)
+    fn signature(&self) -> String {
+        String::from_utf8(Vec::from(&self.header.sig)).unwrap()
```

Comment:
Since this is dead code, I think we can afford changing the signature:
```suggestion
    fn signature(&self) -> Result<String, FromUtf8Error> {
        String::from_utf8(Vec::from(&self.header.sig))
```

Reply:
Good idea. I'll make that change.